### PR TITLE
Shrink permissions of vc scheduler & controller

### PR DESCRIPTION
**Pull request by Monokaix**
Part of https://github.com/volcano-sh/volcano/issues/3505

### CODE REVIEW

File: installer/helm/chart/volcano/templates/scheduler.yaml [hunk 1]
```diff
@@ -52,8 +52,11 @@ rules:
     resources: ["events"]
     verbs: ["create", "list", "watch", "update", "patch"]
   - apiGroups: [""]
-    resources: ["pods", "pods/status"]
-    verbs: ["create", "get", "list", "watch", "update", "patch", "bind", "updateStatus", "delete"]
```
[the remark this answers is not in the document]
Comment (Monokaix) [reply installer/helm/chart/volcano/templates/scheduler.yaml:56]: binding is a subsresource and we can set its rule seperately, line 61 in this file has defined that.

[the remark this answers is not in the document]
Comment (Monokaix) [reply installer/helm/chart/volcano/templates/scheduler.yaml:56]: some detail: https://github.com/kubernetes/kubernetes/issues/75749，binding verb has deprecated in v1.7.

[the remark this answers is not in the document]
Comment (googs1025) [reply installer/helm/chart/volcano/templates/scheduler.yaml:56]: thank for reminding. got it

**Review by hwdef — COMMENTED** [review]
/lgtm

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**